robot_ws/src/voice_interaction_robot/src/voice_interaction_robot/voice_input.py

```diff
@@ -16,16 +15,17 @@
  SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 """
 
-import rospy
-from audio_common_msgs.msg import AudioData
 import pyaudio
 import audioop
 import numpy as np
 import math
 from collections import deque
 
+import rclpy
+from rclpy.node import Node
+from voice_interaction_robot_msgs.msg import AudioData
 
-class VoiceInput:
+class VoiceInput(Node):
     p = pyaudio.PyAudio()
```

Comment:
Don't need to change since this is the same in ROS1, but `p` isn't a super helpful variable name